src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -147,16 +146,41 @@ export class OsmosisSwapper implements Swapper<ChainId> {
         }),
       )
 
+    const sellAssetIsOnOsmosisNetwork = sellAsset.chainId === osmosisChainId
+    const cosmosFees = await cosmosAdapter.getFeeData(getFeeDataInput)
+    const osmoFees = await osmosisAdapter.getFeeData(getFeeDataInput)
```

Comment:
already fetching osmosis fees on line 138, can remove that in favor of this addition